prime-router/docs/onboarding-users/senders.md

```diff
@@ -63,6 +63,30 @@ The schema provides additional transforms required so that the senders data can
 
 Note: When creating a transform, please reference [Changing/Updating Sender/Receiver Transforms](../getting-started/standard-operating-procedures/changing-transforms.md) for guidance.
 
+### Authenticating to ReportStream’s REST API
+**Note: This is legacy authentication and should not be used for onboarding new users**
+
+Shared secret key authorization
+
+Prior to connecting to the endpoint, you’ll need a public/private keypair. There are many ways to do this. The steps below show how to create a key pair using `openssl`.
+
+EC
+```
+openssl ecparam -genkey -name secp384r1 -noout -out my-es-keypair.pem
+openssl ec -in my-es-keypair.pem -pubout -out  my-es-public-key.pem
+```
+
+RSA
+```
+openssl genrsa -out my-rsa-keypair.pem 2048
+openssl rsa -in my-rsa-keypair.pem -outform PEM -pubout -out my-rsa-public-key.pem
+```
+
+Send the public key to the ReportStream team using [our public key tool](/manage-public-key).
+Note: you’ll need to login to use that feature. If you do not have a login contact ReportStream support at [reportstream@cdc.gov](mailto:reportstream@cdc.gov). ReportStream will associate the key with your configuration within ReportStream.
+
+You only need to do this step once, not every time you submit reports. If you need to change your keys at any time, contact ReportStream support.
+
```

Comment:
@GilmoreA6  Do senders need to upload a key even if they're using a shared secret? I thought that was only needed for authenticating using JWT.

Reply:
@arnejduranovic we removed these instructions from RS (/developer-resources/api) and wanted to move them here. Are you suggesting that they are not needed and to close this PR?

Reply:
The ticket Penny is working on was removing the shared secret auth method from the website and the developer's guide. I asked that the examples from the developer's guide for shared secret be added here for our own engineers as there are still legacy senders used shared secret auth and we should have that documentation for ourselves (yes theoretically that means since we have a public repo someone could figure out that it's an option by reading the document but they could do that by looking at the code anyway).

Reply:
@victor-chaparro to answer your question no, it's one or the other.

Reply:
So we should remove that section from here. Since we're only documenting the shared secret piece?

Reply:
There's an authorization section further down the document that I think both examples would fit into

Reply:
Ok, I thought we were only documenting share secret authorization here, but it doesn't hurt to add both.